packages/data/platform/users_flutter/lib/src/storage_json.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+
+import 'package:json_annotation/json_annotation.dart';
+import 'package:storage_access/storage_access.dart' as storage;
+import 'package:users/users.dart';
+
+part 'storage_json.g.dart';
+
+String _storageFileName = "local_users.json";
+
+@JsonSerializable()
+class UsersStorageJson implements Users {
+  final List<User> users;
+
+  UsersStorageJson({List<User> users}) : this.users = users ?? [];
+
+  Map<String, Object> toJson() => _$UsersStorageJsonToJson(this);
+
+  static UsersStorageJson fromJson(Map<String, Object> json) =>
+      _$UsersStorageJsonFromJson(json);
+
+  Future<bool> write() async => storage.write(
+        data: json.encode(this.toJson()),
+        asFile: _storageFileName,
+      );
+
+  Future<bool> prepare() async {
+    try {
+      final readed = await storage.read(fromFile: _storageFileName);
+      if (readed != null) {
+        this
+            .users
```

Comment:
Nit: `.users` might go to prev line

Reply:
Ouch. I use some beautify-code packages for dart and flutter. I guess, they wanted it.